WIP-CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
+### Release notes
```

Comment:
All this content should be in the `CHANGELOG.md` and this should have been resetted, I guess step 2 in this readme was missed before creating this release branch https://github.com/elastisys/compliantkubernetes-kubespray/blob/main/release/README.md#new-kubespray-releases